personalities/sctfe/handlers.go

```diff
@@ -326,21 +331,36 @@ func addChainInternal(ctx context.Context, li *logInfo, w http.ResponseWriter, r
 		return http.StatusBadRequest, fmt.Errorf("failed to build MerkleTreeLeaf: %s", err)
 	}
 
-	// TODO(phboneff): refactor entryFromChain to avoid recomputing hashes in AddIssuerChain
-	if len(chain) > 1 {
+	klog.V(2).Infof("%s: %s => storage.GetCertIndex", li.LogOrigin, method)
+	idx, ok, err := li.storage.GetCertIndex(ctx, chain[0])
+	if err != nil {
+		return http.StatusInternalServerError, fmt.Errorf("couldn't deduplicate the request: %s", err)
```

Comment:
The `error` returned in this `addChainInternal` func is not consistent (`%s` vs `%v`).

Reply:
Yes, I'm aware, but was not intending to fix this in this PR without understanding further why this file uses both in the first place. That's already the case with the CTFE. I presume that it's just oversight, but there might be a specific reason regarding how we displays errors to users. There's already a TODO somewhere to do an error overall review.